docs/signature/README.md

```diff
@@ -0,0 +1,228 @@
+    ]
+  }
+}
+```
+
+### Signature Persisted within an OCI Artifact Enabled Registry
+
+Both values are persisted in a `signature.json` file. The file would be submitted to a registry as an Artifact with null layers.
+The `signature.json` would be persisted wthin the `manifest.config` object
+
+``` SHELL
+oras push \
+  registry.example.com/hello-world:v1.0 \
+  --manifest-config signature.json:application/vnd.cncf.notary.config.v2+json
+```
+
+Would push the following manifest:
+
+``` JSON
+{
+  "schemaVersion": 2,
+  "config": {
+    "mediaType": "application/vnd.cncf.notary.config.v2+json",
+    "size": 1906,
+    "digest": "sha256:c7848182f2c817415f0de63206f9e4220012cbb0bdb750c2ecf8020350239814"
+  },
+  "layers": []
+}
+```
+
+## *Signature* Property Descriptions
+
+- **`signed`** *object*
+
+  This REQUIRED property provides the signed content.
+
+  - **`iat`** *integer*
+
+    This OPTIONAL property identities the time at which the manifests were presented to the notary. This field is based on [RFC 7519 Section 4.1.6](https://tools.ietf.org/html/rfc7519#section-4.1.6). When used, it does not imply the issue time of any signature in the `signatures` property.
+
+  - **`nbf`** *integer*
+
+    This OPTIONAL property identifies the time before which the  signed content MUST NOT be accepted for processing. This field is based on [RFC 7519 Section 4.1.5](https://tools.ietf.org/html/rfc7519#section-4.1.5).
+
+  - **`exp`** *integer*
+
+    This OPTIONAL property identifies the expiration time on or after which the signed content MUST NOT be accepted for processing. This field is based on [RFC 7519 Section 4.1.4](https://tools.ietf.org/html/rfc7519#section-4.1.4).
+
+  - **`digest`** *string*
+
+    This REQUIRED property is the *digest* of the target manifest, conforming to the requirements outlined in [Digests](https://github.com/opencontainers/image-spec/blob/master/descriptor.md#digests). If the actual content is fetched according to the *digest*, implementations MUST verify the content against the *digest*.
```

Comment:
I think it would be good to explicitly add that the digest must be compared to the contents of the image, to make sure that the image actually matches the digest that was signed.